Disable the moz-overlay-scrollbars, as it causes Komodo to consume 20% CPU
whilst sitting idle - bug 100457.

Index: /Users/toddw/as/komodo-devel_moz24/mozilla/build/moz2400-ko8.11/mozilla/toolkit/themes/osx/global/nativescrollbars.css
--- a/toolkit/themes/osx/global/nativescrollbars.css	Tue Sep 10 18:37:39 2013 -0400
+++ b/toolkit/themes/osx/global/nativescrollbars.css	Wed Sep 25 09:49:31 2013 -0700
@@ -13,10 +13,11 @@
   pointer-events: auto;
 }
 
+/* XXX Komodo: Disable this for 8.5.x due to CPU usage issues, bug 100457
 @media all and (-moz-overlay-scrollbars) {
   scrollbar[root="true"] {
     position: relative;
-    z-index: 2147483647; /* largest positive value of a signed 32-bit integer */
+    z-index: 2147483647; /* largest positive value of a signed 32-bit integer *
   }
 
   scrollbar:not([active="true"]),
@@ -24,6 +25,7 @@
     visibility: hidden;
   }
 }
+*/
 
 html|select[size]:not([size="0"]):not([size="1"]) > scrollbar,
 html|select[multiple] > scrollbar {
